examples_gbs/run_tutorial_dense.py

```diff
@@ -0,0 +1,120 @@
+# graph is generated with edge probability of 0.875. The planted nodes are the last ten nodes in the
+# graph. The two graphs are joined by selecting 8 vertices at random from both graphs and adding an
+# edge between them. This graph has the sneaky property that even though the planted subgraph is the
+# densest of its size, its nodes have a lower average degree than the nodes in the rest of the
+# graph.
+#
+# The :mod:`~.gbs.data` module has pre-generated GBS samples from this graph. Let's load them,
+# postselect on samples with a large number of clicks, and convert them to subgraphs:
+
+planted = data.Planted()
+postselected = sample.postselect(planted, 16, 30)
+pl_graph = nx.to_networkx_graph(planted.adj)
+samples = sample.to_subgraphs(postselected, pl_graph)
+print(len(samples))
+
+##############################################################################
+# Not bad! We have more than 2000 samples to play with. The planted subgraph is actually easy to
+# identify; it evens appears clearly from the force-directed Kamada-Kawai algorithm that is used to
+# plot graphs in Strawberry Fields:
+sub = list(range(20, 30))
+plot_graph = plot.plot_graph(pl_graph, sub)
+plotly.offline.plot(plot_graph, filename="planted.html")
+
+##############################################################################
+# .. raw:: html
+#     :file: ../../examples_gbs/planted.html
+#
+# .. note::
+#     The command ``plotly.offline.plot()`` is used to display plots in the documentation. In
+#     practice, you can simply use ``plot_graph.show()`` to view the graph.
+
+##############################################################################
+# A more interesting challenge is to find dense subgraphs of different sizes; it is often
+# useful to identify many high-density subgraphs, not just the densest ones. This is the purpose of
+# the :func:`~.search` function in the :mod:`~gbs.subgraph` module: to identify collections of dense
+# subgraphs for a range of sizes. The output of this function is a dictionary whose keys correspond
+# to subgraph sizes within the specified range. The items in the dictionary are the top subgraphs of
+# that size and their corresponding density.
+
+dense = subgraph.search(samples, pl_graph, 8, 16, max_count=3)  # we look at top 3 densest subgraphs
+for k in range(8, 17):
+    print(dense[k][0])  # print only the densest subgraph of each size
+
+##############################################################################
+# From the results of the search we learn that, depending on their size, the densest subgraphs
+# belong to different regions of the graph: dense subgraphs of less than ten nodes are contained
+# within the planted subgraph, whereas larger dense subgraphs appear outside of the planted
+# subgraph. Smaller dense subgraphs can be cliques, characterized by having
+# maximum density of 1, while larger subgraphs are less dense. Let's see what the smallest and
+# largest subgraphs look like:

```

Comment:
Very cool!!!

Reply:
:grin: I think this fixes the "issue" of the plot being able to identify the planted subgraph